src/solver/conv_mlir_igemm_bwd.cpp

```diff
@@ -59,29 +47,56 @@ bool ConvMlirIgemmBwd::IsApplicable(const ConvolutionContext& ctx) const
     if(!IsComposableKernelSupportedHardware(ctx))
         return false;
 
-    return MiirIsConfigApplicable(
-        mlir::ConstructBuildOptions(ctx, GetOperation(), GetKernelName(), false));
+    return MiirIsConfigApplicable(mlir::ConstructBuildOptions(ctx, false));
 #else
     std::ignore = ctx;
     return false;
 #endif
 }
 
-ConvSolution ConvMlirIgemmBwd::GetSolution(const ConvolutionContext& ctx) const
+PerformanceConvMlirIgemm ConvMlirIgemmBwd::GetPerformanceConfig(const ConvolutionContext& ctx) const
+{
+    std::ignore = ctx;
+    return {};
+}
```

Comment:
This produces invalid performance config, which does not match the [specification](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/866#issuecomment-822585406). Why [this](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1152/files#r707509009) is done in `GetSoluiton` but not in this function?

Reply:
Ping @jerryyin to check. Sorry to have missed it in last review.

Reply:
I agree it break the existing specification, but this is intentional. I will explain why I made this decision in #1154 in case this thread becomes too long. Then based on the discussion result, we can decide what's the best behavior it is for mlir solvers.